app/components/bd-map.js

```diff
@@ -1,4 +1,5 @@
-import Leaflet from 'bus-detective/utils/leaflet';
```

Comment:
I created this wrapper module so we're aren't referencing the window variable. Is there any reason we can't use that?